src/app/Dashboard/LayoutTemplatePicker.tsx

```diff
@@ -0,0 +1,638 @@
+                          </EmptyState>
+                        ) : (
+                          <>
+                            {selectedTemplate.cards
+                              .slice(0, CARD_PREVIEW_LIMIT)
+                              .filter((cfg) => hasConfigByName(cfg.name))
+                              .map((cfg, idx) => (
+                                <GridItem span={numCards === 1 ? 12 : 6} key={idx} order={{ default: idx.toString() }}>
+                                  {/* TODO: remove this once we have a preview for JFRMetricsChartCard */}
+                                  {cfg.name === 'JFRMetricsChartCard' ? (
+                                    <Card isFullHeight isCompact>
+                                      <CardHeader>
+                                        <Title headingLevel={'h4'}>{cfg.props['chartKind']}</Title>
+                                      </CardHeader>
+                                      <CardBody>
+                                        <Bullseye>
+                                          <EmptyState>
+                                            <EmptyStateBody>Empty preview</EmptyStateBody>
+                                          </EmptyState>
+                                        </Bullseye>
+                                      </CardBody>
+                                    </Card>
+                                  ) : (
+                                    <div className="preview-card">
+                                      {React.createElement(getConfigByName(cfg.name).component, {
+                                        span: cfg.span,
+                                        ...cfg.props,
+                                        dashboardId: idx,
+                                        actions: [],
+                                        isDraggable: false,
+                                        isResizable: false,
+                                        isFullHeight: false,
+                                      })}
+                                    </div>
+                                  )}
+                                </GridItem>
+                              ))}
+                            {numCards > CARD_PREVIEW_LIMIT && (
+                              <GridItem
+                                span={12}
+                                key={CARD_PREVIEW_LIMIT}
+                                order={{ default: CARD_PREVIEW_LIMIT.toString() }}
+                              >
+                                <Card isFullHeight isCompact>
+                                  <CardHeader>
+                                    <Title headingLevel={'h4'}>Remaining cards ({numCards - CARD_PREVIEW_LIMIT})</Title>
+                                  </CardHeader>
+                                  <CardBody>
+                                    <List variant={ListVariant.inline} isBordered isPlain>
```

Comment:
Maybe no border? Right now looks like last item is missing an underline.

![Screenshot from 2023-04-12 22-03-01](https://user-images.githubusercontent.com/68053619/231627951-87ec46ee-1426-4d2f-9e24-99b3e6cd4c98.png)

Reply:
Remove border:

![Screenshot from 2023-04-12 22-04-52](https://user-images.githubusercontent.com/68053619/231628046-b6fc22bc-4b17-484f-82b2-149a48547f5c.png)